style.css

```diff
@@ -83,12 +89,12 @@ main {
 }
 
 .work-experience-part-one-description {
-    width: 45%;
+    width: 300px;
     font-size: 20px;
 }
 
 .work-experience-part-one-list {
-    width: 100%;
+    width: 104%;
```

Comment:
Setting an element's width to more than 100% might push other elements out of their expected positions or cause unexpected wrapping or scaling. I'd advise against it